### PR TITLE
Bump Kubernetes to v1.20.0-alpha.1

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind dependency-change

#### What this PR does / why we need it:
Update Kubernetes to the latest alpha release.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -54,14 +54,14 @@ require (
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1
 	google.golang.org/grpc v1.30.0
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.18.6
+	k8s.io/api v0.19.2
+	k8s.io/apimachinery v0.19.2
```
Comment (TomSweeneyRedHat) [inline go.mod:58]: Title says 1.20.0-alpha1?

Comment (umohnani8) [reply go.mod:58]: @saschagrunert any idea why the version looks off in go.mod?

**Comment by umohnani8** [comment]
A question above, otherwise LGTM

File: go.mod (hunk 1 above)
Comment (saschagrunert) [reply go.mod:58]: We overwrite the version below, I do not have any influence on this version selection :D

**Comment by umohnani8** [comment]
Let's get this in
/lgtm

**Pull request merged**